README.md

```diff
@@ -1,6 +1,42 @@
 # edugator-staff-frontend
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+An interactive coding and learning portal for students in STEM.
+
+Need info? Check the Docs | Create an Issue | Ask us on Discord
```

Comment:
Would be good to add the discord join link somewhere.

Reply:
I'll add to a later PR

Reply:
Sounds good.